CHANGELOG.md

```diff
@@ -9,17 +16,21 @@
 * builder/vmware-esx: Add `network_name` option to vmware so that users can set
     a network without using vmx data. [GH-9718]
 * builder/vmware-vmx: Add additional disk configuration option.  Previously
-    only implemented for vmware-iso builder [Gh-9815]
+    only implemented for vmware-iso builder [GH-9815]
 * builder/vmware: Add a `remote_output_directory option` so users can tell
     Packer where on a datastore to create a vm. [GH-9784]
 * builder/vmware: Add option to export to ovf or ova from a local vmware build
     [GH-9825]
 * builder/vmware: Add progress tracker to vmware-esx5 iso upload. [GH-9779]
+* builder/vsphere-iso:  Add support for building on a single ESXi host
```

Comment:
this isn't the right blurb: integrations/secretsmanager: Add support for plaintext, non key/pair, secrets #9773

Reply:
Ah good catch! Looks like I was off by 20 :smile: - it should be 9793.